ecl/mvna/v1/_proxy.py

```diff
@@ -857,48 +853,45 @@ def get_staged_policy_configuration(self, policy_id):
         """
         return self._get(_policy.Policy, policy_id)
 
-    # NOTE(NaoShark): :param sorry_page_url: and :param certificate_id: and
-    # :param tls_security_policy_id: will be available from Day 2.
-
     def update_staged_policy_configuration(self, policy_id,
                                            algorithm=None, persistence=None,
-                                           # sorry_page_url=None,
-                                           # certificate_id=None,
+                                           sorry_page_url=None,
+                                           certificate_id=None,
                                            health_monitor_id=None,
                                            listener_id=None,
                                            default_target_group_id=None,
-                                           # tls_security_policy_id=None
+                                           tls_policy_id=None
                                            ):
         """Update Staged Policy Configuration.
 
         :param string policy_id: ID of Policy
         :param string algorithm: Algorithm of Policy
         :param string persistence: Persistence of Policy
-        # :param string sorry_page_url: Sorry page URL
-        # :param string certificate_id: Certificate ID
+        :param string sorry_page_url: Sorry page URL
+        :param string certificate_id: Certificate ID
         :param string health_monitor_id: Health Monitor ID of Policy
         :param string listener_id: Listener ID of Policy
         :param string default_target_group_id: Default Target Group ID
-        # :param string tls_security_policy_id: TLS Security Policy ID
+        :param string tls_policy_id: TLS Policy ID
         :return: Policy
         """
         body = {}
         if algorithm:
             body["algorithm"] = algorithm
         if persistence:
             body["persistence"] = persistence
-        # if sorry_page_url:
-        #     body["sorry_page_url"] = sorry_page_url
-        # if certificate_id:
-        #     body["certificate_id"] = certificate_id
+        if sorry_page_url:
```

Comment:
```suggestion
        if sorry_page_url is not None:
```
この実装だと、空文字列に変更ができません。他の箇所にも同様に適用をお願いします。